sandbox-backend/src/main/resources/db/migration/V0000000__initial.sql

```diff
@@ -80,11 +80,15 @@ CREATE TABLE discount_tag
     FOREIGN KEY (tagId) REFERENCES tag (id)
 );
 
-CREATE TABLE employee_discount
```

Comment:
Don't modify existing script for incremental migration. Add new migration with table renaming and new fields (ALTER TABLE ...)

Reply:
Done.